x-pack/legacy/plugins/lens/public/editor_frame_service/embeddable/embeddable_factory.ts

```diff
@@ -27,17 +27,19 @@ import { Embeddable } from './embeddable';
 import { SavedObjectIndexStore, DOC_TYPE } from '../../persistence';
 import { getEditPath } from '../../../../../../plugins/lens/common';
 
+interface StartServices {
```

Comment:
Not relevant for this PR, but this object is a great example of how inconsistent our naming is 
`xxxContract`,`xxxType`,`xxxSetup` and a type with no suffix at all - in the same object :)
I guess we should choose and stick to one or two.

Reply:
agree!  I called this `startServices` based off the naming of `core.getStartServices()`.